### ISSUE TITLE
Ecrire un parser de poids et biais et de donnee de test

### ISSUE THREAD

**Issue by LilianSchall** [LilianSchall/SciKitPlusPlus#3]
# Quoi ?

En prenant en entree un fichier texte au format de votre choix, reussir a extraire les nombres flottants qui y sont contenus.

Pour le fichier texte de modele, il faut arriver a extraire pour le moment deux vecteurs de flottants: les poids et les biais, puis d'instancier un objet `Linear` avec ces vecteurs.

Pour le fichier texte de dataset, il faut arriver a extraire la matrice de donnees stockees. 

# Pourquoi ?

Parce que le ticket #1 nous permet de generer des modeles simples de regression lineaire et on souhaite savoir si l'on arrive a les charger en memoire et avoir la meme accuracy qu'en sklearn.

# Ou ?

Un dossier loader va etre cree avec deux couples "source-header", un couple par parser. La structure est libre donc implementez ce que vous voulez (soit une classe soit une fonction, meme si une fonction devrait suffire en verite)

**Comment by LilianSchall** [comment]
Closed by #5 .
